test/unit/graphql-config.test.js

```diff
@@ -0,0 +1,35 @@
+import { expect, jest } from '@jest/globals'
```

Comment:
update these tests to reflect the new name for the config value `enableIntrospection` or w/e you decide